src/riscv-integration.adoc

```diff
@@ -507,7 +507,7 @@ its address incremented by four times the interrupt cause number.
 
 Capabilities written to <<mtvecc>> also include writing the MODE field in
 **mtvecc.address[1:0]**. As a result, a representability and sealing check is
-performed on the capability with the legalized (WARL) MODE field included in
+performed on the capability with the MODE field zeroed in
```

Comment:
Hmm I'm not sure about this one - ISAv9 Qemu does the check with the legalized mode. Then again I am not convinced this actually matters since you can use mode==1 with sentries. 

I also feel that we should be a bit more explicit here about what happens if you install a sentry in mtvecc.

Reply:
The point here is that it's possible (in a very bizarre way) to have  a cap which is representable when the LSB of the address is set, and not representable when the LSB is clear, so jumping to MTVECC in this case means jumping to a bad capability. That's the bug I'm trying to fix here, and it implies that Qemu on CHERI v9 has this an an undetected bug.

For sentries in mtvecc - I agree the spec isn't clear - clarifying that should be another PR.

Reply:
ok - updated - including clarifying sentry behaviour

Reply:
V9 wouldn't have this problem - it's because we hade the A<R check use many more bits to in the extreme case even address bit [0] can actually affect the representability result, which was probably an oversight

Reply:
https://github.com/riscv/riscv-cheri/issues/561 opened to track the sentry issue. Otherwise this has been resolved.